### DISCUSSION TITLE
Source file of elementary flows database

### DISCUSSION THREAD

**Discussion by nanocate** [LCA-ActivityBrowser/activity-browser#749] [Q&A]
Hello! 
I was wondering if one can retrieve the biosphere3 database as an excel file? Or could you please share the source where you got this dataset?
Thanks a lot in advance!
Best regards, 
Caterin

**Comment by marc-vdm** [comment]
Sorry for the extremely delayed response. Under `File` in the top left, you can choose `Export Database` and then export the biosphere database as excel file.
